sdk/tlaxcala/Makefile

```diff
@@ -6,18 +6,12 @@ bin/tlaxcala: src/main.cc wabt json
 	mkdir -p bin/
 	g++ src/main.cc -o bin/tlaxcala -I./wabt/ -I./wabt/build -L ./wabt/build -I./json/include -lwabt
 
-.PHONY: wabt
```

Comment:
Aren't `wabt` and `json` Makefile targets still `PHONY`?

Reply:
No, these two targets will check if the dir exists. This removes the rebuild in every build.